classes/PodsAPI.php

```diff
@@ -8681,7 +8681,10 @@ public function get_table_info( $object_type, $object, $name = null, $pod = null
 			if ( ! did_action( 'init' ) ) {
 				$transient .= '_pre_init';
 			}
-			pods_transient_set( $transient, $info );
+
+			if ( false !== $_info && is_array( $_info ) ) {
```

Comment:
I believe this condition has to be reversed.

```php
if ( false === $_info )
```

Because https://github.com/pods-framework/pods/blob/30dfc0a0822d816b1cd7b729e35028af7cc9e9e7/classes/PodsAPI.php#L8295

We need to avoid re-saving the exact same information that was just grabbed from the transient.

We only save, when transient didn't have anything.

Do I understand this correctly?

Reply:
Yeah, I reversed the entire logic in my head, I think the whole thing needs be negated

Reply:
It was introduced here, so I'll assume there is a reason: https://github.com/pods-framework/pods/commit/e9ad5fc7cdcbd63774c7081e38570946a6f74db5